kubernetes/core-pod.yml

```
@@ -23,3 +23,12 @@ spec:
           port: 5000
         initialDelaySeconds: 10
         timeoutSeconds: 40
+      volumeMounts:
+        - name: core-volume
+          mountPath: /app-data/transactions.json
+        - name: core-volume
+          mountPath: /app-data/users.json
```

So sorry for scam. I didn't realize the mountPath param means directory path, not the certain file.

```suggestion
        - name: core-volume
          mountPath: /data/
```